### DISCUSSION TITLE
Links to my animations

### DISCUSSION THREAD

**Discussion by ghost** [community/community#50454] [New to GitHub]
Hi all, I'm new to GitHub a few weeks ago. Just upgraded to the Pro Plan, but I'm having trouble with my Lottie animations that I store here. I'm uploading them, copying the url from the raw page and inserting into the Lottie player in my Squarespace site. All works fine for a few minutes, then link is broken. I notice if I repeat the process the url changes in the raw file though I have not re-uploaded it. I tried this in a brand new website with no other custom code to make sure that wasn't the problem. 

I tried a few things to make sure the problem isn't something else. Animations I copy from GitHub play fine in a Lottie desktop player, and Lottie animations that I take straight from their website are fine when I put them on my Squarespace website.

Can anyone help me with this? I've seen a couple other references to the problem online but no solutions. I've been trying everything I can think of for over a week.

**Comment by SMelidoni** [comment]
Hi there @zeebelt 👋🏽

It sounds like the issue you're experiencing might be related to caching. When you upload the Lottie animation and copy the URL, Squarespace might be caching the file for a certain amount of time, which would explain why it works fine for a few minutes and then stops working. When you repeat the process and the URL changes, it's possible that GitHub is caching the file with a new URL, which Squarespace is then caching again.

One possible solution would be to try using a different hosting service for your Lottie animations, such as a content delivery network (CDN) or a cloud storage service like Amazon S3 or Google Cloud Storage. These services are designed to handle caching and can often be configured to serve files with a longer cache expiration time, which should help prevent the issue you're experiencing.

Alternatively, you could try modifying the URL of the Lottie animation each time you upload a new version of it to GitHub. This would ensure that Squarespace doesn't cache the file for too long, since the URL is different each time. For example, you could add a version number or timestamp to the end of the URL.

If neither of these solutions work, you might want to reach out to Squarespace support or GitHub support for further assistance. They may be able to provide more specific guidance based on your setup and the exact issue you're experiencing.

Hope this helps!
